packages/@cdktf/hcl2json/lib/index.ts

```diff
@@ -114,7 +114,7 @@ export async function convertFiles(
     }
   }
 
-  if (tfFileContents === "" && tfJSONFileContents === []) {
+  if (tfFileContents === "" && tfJSONFileContents.length === 0) {
```

Comment:
whoops – this appears to have broken tests 😅
We can also remove this entire thing 😄 What do you think?

Reply:
Fixed it by returning an empty object as required by the test. Also adjusted the testcase to check that the error is actually printed